cli/go.mod

```diff
@@ -145,3 +145,6 @@ require (
 
 // github.com/cloudquery/jsonschema @ cqmain
 replace github.com/invopop/jsonschema => github.com/cloudquery/jsonschema v0.0.0-20240202134451-d771afde32fb
+
+// github.com/cloudquery/godebouncer @ fix-race
+replace github.com/vnteamopen/godebouncer => github.com/cloudquery/godebouncer v0.0.0-20240220152946-054f7dbbcf18
```

Comment:
The new test uncovered a race condition in the godebouncer library. I'm fixing it with a replace directive here, but there is also an upstream PR https://github.com/vnteamopen/godebouncer/pull/28